[2025-02-22T20:00:03.086+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-22T20:00:03.118+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-22T19:00:00+00:00 [queued]>
[2025-02-22T20:00:03.129+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-22T19:00:00+00:00 [queued]>
[2025-02-22T20:00:03.129+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-22T20:00:03.147+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-22 19:00:00+00:00
[2025-02-22T20:00:03.153+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5624) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-22T20:00:03.154+0000] {standard_task_runner.py:72} INFO - Started process 5625 to run task
[2025-02-22T20:00:03.155+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'london_weather_pipeline', 'transform_weather_data', 'scheduled__2025-02-22T19:00:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/london_weather_pipeline/london_weather_etl.py', '--cfg-path', '/tmp/tmpmws3fx4v']
[2025-02-22T20:00:03.156+0000] {standard_task_runner.py:105} INFO - Job 49: Subtask transform_weather_data
[2025-02-22T20:00:03.245+0000] {task_command.py:467} INFO - Running <TaskInstance: london_weather_pipeline.transform_weather_data scheduled__2025-02-22T19:00:00+00:00 [running]> on host ee2081ae3a04
[2025-02-22T20:00:03.382+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='london_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T19:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-22T19:00:00+00:00'
[2025-02-22T20:00:03.383+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-22T20:00:03.386+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '51.5074', 'longitude': '-0.1278', 'temperature': 10.5, 'windspeed': 8.7, 'winddirection': 228, 'weathercode': 0}
[2025-02-22T20:00:03.425+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-22T20:00:03.426+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=london_weather_pipeline, task_id=transform_weather_data, run_id=scheduled__2025-02-22T19:00:00+00:00, execution_date=20250222T190000, start_date=20250222T200003, end_date=20250222T200003
[2025-02-22T20:00:03.489+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-22T20:00:03.519+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-22T20:00:03.522+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
